src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -34,6 +35,9 @@ class ContainerAwareEventManager extends EventManager
     private $methods = [];
     private $container;
 
+    /**
+     * @param list<string|EventSubscriber|array{string[], string|object}> $subscriberIds List of subscribers, subscriber ids, or [events, listener] tuples
```

Comment:
What `array{string[], string|object}` is supposed to mean?

Reply:
It's an array where key `0` contains `string[]` and key `1` contains `string` or `object`.
ie. 
```php
[['foo', 'bar'], 'listnerId'];
```

Reply:
Oh alright, I thought it was a mistake.
Thanks! :)